pkg/daemon/controller/controller.go

```diff
@@ -589,10 +594,25 @@ func (c *CtrlHub) iptablesSyncLoop() {
 			}
 
 			iptablesManager := c.getIPtablesManager(subnet.Spec.Range.Version)
+
+			// isLocal means whether this node belongs to this network
+			isLocal := nodeBelongsToNetwork(c.config.NodeName, network)
+
+			// if network is local vlan, record vlan forward interface names
+			if isLocal && networkingv1.GetNetworkMode(network) == networkingv1.NetworkModeVlan {
+				netID := subnet.Spec.NetID
+				if netID == nil {
+					netID = network.Spec.NetID
+				}
+
+				if vlanForwardIfName, err := daemonutils.GenerateVlanNetIfName(c.config.NodeVlanIfName, netID); err == nil {
```

Comment:
Could you please log the err if it's not nil?

Reply:
fixed